packages/python/README.md

```
@@ -0,0 +1,125 @@
+
+> [!WARNING]
+> `envoic` is still experimental and therefore subject to major changes across releases. Breaking changes may occur until `v1.0.0`.
+
+
+
+# Use Immediately
+
+Run without installing:
+
+```bash
+$ uvx envoic scan .
```

markdownlint-cli2 (0.21.0)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)